scripts/keybindings/2 normal_mode.cjs

```diff
@@ -1,11 +1,10 @@
-const { key2arg, and, addKeybinds, or } = require("./util.cjs");
+const { key2arg, and, addKeybinds } = require("./util.cjs");
 
 const [add, keybinds] = addKeybinds();
 
-const when = or(
```

Comment:
If you're just going to do this anyway, probably not worth the boilerplate to have and/or.

Reply:
Um. Actually, when I was writing, I also thought about whether it was necessary, but considering it's just a script, I was quite casual about it.

Reply:
Removed